whois-commons/src/main/java/net/ripe/db/whois/common/SentryConfigurations.java

```diff
@@ -0,0 +1,44 @@
+package net.ripe.db.whois.common;
+
+import io.sentry.Sentry;
+import org.apache.commons.lang.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+
+@Component
+public class SentryConfigurations {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(SentryConfigurations.class);
+
+    private final ApplicationVersion applicationVersion;
+    private final String sentryDsn;
+    private final String environment;
+
+    @Autowired
+    public SentryConfigurations(
+            @Value("${sentry.dsn:}") final String sentryDsn,
+            @Value("${sentry.environment:}") final String environment,
```

Comment:
`sentry.environment` is optional, should you check for empty string and return if not defined? Or should we assume a default value (such as production)?

Reply:
Ok I will add a check I guess making default as production is error prone as if someone forgets to add environment property in predev errors will be reported as production then

Reply:
Done

Reply:
Thanks!